build/linux/flavorgen/schema/common.schema.json

```diff
@@ -0,0 +1,105 @@
+{
```

Comment:
Don't duplicate the schemas.  Just copy the same file from build/linux/schemas when creating the installer package.